batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/util/DbConnection.java

```diff
@@ -0,0 +1,39 @@
+package com.lftechnology.batch7crud.util;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by pratishshr on 1/14/16.
+ */
+public class DbConnection {
+    private static Logger logger = Logger.getLogger("DbConnectionLog");
+
+    private DbConnection() {
+
+    }
+
+    public static Connection getConnection() throws DataException {
+        try {
+            Context initCtx = new InitialContext();
+            Context envCtx = (Context) initCtx.lookup("java:comp/env");
+            DataSource ds = (DataSource) envCtx.lookup("jdbc/ems");
+
+            return ds.getConnection();
+
+        } catch (SQLException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)